docker/Dockerfile

```diff
@@ -16,12 +16,8 @@ RUN pip install --no-cache-dir --upgrade pip && \
     pip install --no-cache-dir -r /app/requirements.txt && \
     rm -rf ~/.cache/pip
 
-ADD ./pynvml /app/pynvml
-ADD ./setup.py /app/
-ADD ./setup.cfg /app/
+ADD ./pynvml_utils /app/pynvml_utils
 ADD ./README.md /app/
-ADD ./PKG-INFO /app/
-ADD ./MANIFEST.in /app/
 ADD ./help_query_gpu.txt /app/
 ADD docker/entrypoint.sh /docker-entrypoint.sh
```

Comment:
Think we can consolidate this to one `ADD` of the whole directory and then use `pip install .`

Alternatively this might be a good time to clean this up if it is unneeded

Reply:
Okay, gave this a try.